https://github.com/pravins/lohit/pull/84.patch

From cf7454ba34b6373166112804fa8122bc88d13311 Mon Sep 17 00:00:00 2001
From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Thu, 25 May 2017 07:40:24 -0400
Subject: [PATCH] Fix appstream metadata

The provided font name must match the font's "fullname" as seen by fc-query
---
 lohit-master/assamese/io.pagure.lohit.assamese.font.metainfo.xml           | 4 ++--
 lohit-master/bengali/io.pagure.lohit.bengali.font.metainfo.xml             | 4 ++--
 lohit-master/devanagari/io.pagure.lohit.devanagari.font.metainfo.xml       | 4 ++--
 lohit-master/gujarati/io.pagure.lohit.gujarati.font.metainfo.xml           | 4 ++--
 lohit-master/gurmukhi/io.pagure.lohit.gurmukhi.font.metainfo.xml           | 4 ++--
 lohit-master/kannada/lohit-master/io.pagure.lohit.kannada.font.metainfo.xml             | 4 ++--
 lohit-master/malayalam/io.pagure.lohit.malayalam.font.metainfo.xml         | 4 ++--
 lohit-master/marathi/io.pagure.lohit.marathi.font.metainfo.xml             | 4 ++--
 lohit-master/nepali/io.pagure.lohit.nepali.font.metainfo.xml               | 4 ++--
 lohit-master/odia/lohit-master/io.pagure.lohit.odia.font.metainfo.xml                   | 4 ++--
 lohit-master/.../io.pagure.lohit.tamil.classical.font.metainfo.xml         | 4 ++--
 lohit-master/tamil/io.pagure.lohit.tamil.font.metainfo.xml                 | 4 ++--
 lohit-master/telugu/io.pagure.lohit.telugu.font.metainfo.xml               | 4 ++--
 13 files changed, 26 insertions(+), 26 deletions(-)

diff --git a/lohit-master/assamese/io.pagure.lohit.assamese.font.metainfo.xml b/lohit-master/assamese/io.pagure.lohit.assamese.font.metainfo.xml
index bdb6319..fbae54f 100644
--- a/lohit-master/assamese/io.pagure.lohit.assamese.font.metainfo.xml
+++ b/lohit-master/assamese/io.pagure.lohit.assamese.font.metainfo.xml
@@ -17,9 +17,9 @@
   </description>
   <url type="homepage">https://pagure.io/lohit</url>
   <url type="bugtracker">https://github.com/pravins/lohit/issues</url>
-  <updatecontact>psatpute_at_redhat_dot_com</updatecontact>
+  <update_contact>psatpute_at_redhat_dot_com</update_contact>
   <provides>
-	  <font>Lohit Assamese Regular</font>
+	  <font>Lohit Assamese</font>
   </provides>
   <languages>
 	  <lang>as</lang>
diff --git a/lohit-master/bengali/io.pagure.lohit.bengali.font.metainfo.xml b/lohit-master/bengali/io.pagure.lohit.bengali.font.metainfo.xml
index 78109d2..6a3928d 100644
--- a/lohit-master/bengali/io.pagure.lohit.bengali.font.metainfo.xml
+++ b/lohit-master/bengali/io.pagure.lohit.bengali.font.metainfo.xml
@@ -17,9 +17,9 @@
   </description>
   <url type="homepage">https://pagure.io/lohit</url>
   <url type="bugtracker">https://github.com/pravins/lohit/issues</url>
-  <updatecontact>psatpute_at_redhat_dot_com</updatecontact>
+  <update_contact>psatpute_at_redhat_dot_com</update_contact>
   <provides>
-	  <font>Lohit Bengali Regular</font>
+	  <font>Lohit Bengali</font>
   </provides>
   <languages>
 	  <lang>bn</lang>
diff --git a/lohit-master/devanagari/io.pagure.lohit.devanagari.font.metainfo.xml b/lohit-master/devanagari/io.pagure.lohit.devanagari.font.metainfo.xml
index 0ab8b6b..8fb6a22 100644
--- a/lohit-master/devanagari/io.pagure.lohit.devanagari.font.metainfo.xml
+++ b/lohit-master/devanagari/io.pagure.lohit.devanagari.font.metainfo.xml
@@ -17,9 +17,9 @@
   </description>
   <url type="homepage">https://pagure.io/lohit</url>
   <url type="bugtracker">https://github.com/pravins/lohit/issues</url>
-  <updatecontact>psatpute_at_redhat_dot_com</updatecontact>
+  <update_contact>psatpute_at_redhat_dot_com</update_contact>
   <provides>
-	  <font>Lohit Devanagari Regular</font>
+	  <font>Lohit Devanagari</font>
   </provides>
   <languages>
 	  <lang>hi</lang>
diff --git a/lohit-master/gujarati/io.pagure.lohit.gujarati.font.metainfo.xml b/lohit-master/gujarati/io.pagure.lohit.gujarati.font.metainfo.xml
index cc3f1f4..a005265 100644
--- a/lohit-master/gujarati/io.pagure.lohit.gujarati.font.metainfo.xml
+++ b/lohit-master/gujarati/io.pagure.lohit.gujarati.font.metainfo.xml
@@ -17,9 +17,9 @@
   </description>
   <url type="homepage">https://pagure.io/lohit</url>
   <url type="bugtracker">https://github.com/pravins/lohit/issues</url>
-  <updatecontact>psatpute_at_redhat_dot_com</updatecontact>
+  <update_contact>psatpute_at_redhat_dot_com</update_contact>
   <provides>
-	  <font>Lohit Gujarati Regular</font>
+	  <font>Lohit Gujarati</font>
   </provides>
   <languages>
 	  <lang>gu</lang>
diff --git a/lohit-master/gurmukhi/io.pagure.lohit.gurmukhi.font.metainfo.xml b/lohit-master/gurmukhi/io.pagure.lohit.gurmukhi.font.metainfo.xml
index e350c5e..6effc5b 100644
--- a/lohit-master/gurmukhi/io.pagure.lohit.gurmukhi.font.metainfo.xml
+++ b/lohit-master/gurmukhi/io.pagure.lohit.gurmukhi.font.metainfo.xml
@@ -17,9 +17,9 @@
   </description>
   <url type="homepage">https://pagure.io/lohit</url>
   <url type="bugtracker">https://github.com/pravins/lohit/issues</url>
-  <updatecontact>psatpute_at_redhat_dot_com</updatecontact>
+  <update_contact>psatpute_at_redhat_dot_com</update_contact>
   <provides>
-	  <font>Lohit Gurmukhi Regular</font>
+	  <font>Lohit Gurmukhi</font>
   </provides>
   <languages>
 	  <lang>pa</lang>
diff --git a/lohit-master/kannada/io.pagure.lohit.kannada.font.metainfo.xml b/lohit-master/kannada/io.pagure.lohit.kannada.font.metainfo.xml
index 680b756..8d2ee89 100644
--- a/lohit-master/kannada/io.pagure.lohit.kannada.font.metainfo.xml
+++ b/lohit-master/kannada/io.pagure.lohit.kannada.font.metainfo.xml
@@ -17,9 +17,9 @@
   </description>
   <url type="homepage">https://pagure.io/lohit</url>
   <url type="bugtracker">https://github.com/pravins/lohit/issues</url>
-  <updatecontact>psatpute_at_redhat_dot_com</updatecontact>
+  <update_contact>psatpute_at_redhat_dot_com</update_contact>
   <provides>
-	  <font>Lohit Kannada Regular</font>
+	  <font>Lohit Kannada</font>
   </provides>
   <languages>
 	  <lang>kn</lang>
diff --git a/lohit-master/malayalam/io.pagure.lohit.malayalam.font.metainfo.xml b/lohit-master/malayalam/io.pagure.lohit.malayalam.font.metainfo.xml
index 098d7de..b4e95ca 100644
--- a/lohit-master/malayalam/io.pagure.lohit.malayalam.font.metainfo.xml
+++ b/lohit-master/malayalam/io.pagure.lohit.malayalam.font.metainfo.xml
@@ -17,9 +17,9 @@
   </description>
   <url type="homepage">https://pagure.io/lohit</url>
   <url type="bugtracker">https://github.com/pravins/lohit/issues</url>
-  <updatecontact>psatpute_at_redhat_dot_com</updatecontact>
+  <update_contact>psatpute_at_redhat_dot_com</update_contact>
   <provides>
-	  <font>Lohit Malayalam Regular</font>
+	  <font>Lohit Malayalam</font>
   </provides>
   <languages>
 	  <lang>ml</lang>
diff --git a/lohit-master/marathi/io.pagure.lohit.marathi.font.metainfo.xml b/lohit-master/marathi/io.pagure.lohit.marathi.font.metainfo.xml
index 6a4ea9a..9c8d085 100644
--- a/lohit-master/marathi/io.pagure.lohit.marathi.font.metainfo.xml
+++ b/lohit-master/marathi/io.pagure.lohit.marathi.font.metainfo.xml
@@ -17,9 +17,9 @@
   </description>
   <url type="homepage">https://pagure.io/lohit</url>
   <url type="bugtracker">https://github.com/pravins/lohit/issues</url>
-  <updatecontact>psatpute_at_redhat_dot_com</updatecontact>
+  <update_contact>psatpute_at_redhat_dot_com</update_contact>
   <provides>
-	  <font>Lohit Marathi Regular</font>
+	  <font>Lohit Marathi</font>
   </provides>
   <languages>
 	  <lang>mr</lang>
diff --git a/lohit-master/nepali/io.pagure.lohit.nepali.font.metainfo.xml b/lohit-master/nepali/io.pagure.lohit.nepali.font.metainfo.xml
index 1762764..cd7dfd4 100644
--- a/lohit-master/nepali/io.pagure.lohit.nepali.font.metainfo.xml
+++ b/lohit-master/nepali/io.pagure.lohit.nepali.font.metainfo.xml
@@ -17,9 +17,9 @@
   </description>
   <url type="homepage">https://pagure.io/lohit</url>
   <url type="bugtracker">https://github.com/pravins/lohit/issues</url>
-  <updatecontact>psatpute_at_redhat_dot_com</updatecontact>
+  <update_contact>psatpute_at_redhat_dot_com</update_contact>
   <provides>
-	  <font>Lohit Nepali Regular</font>
+	  <font>Lohit Nepali</font>
   </provides>
   <languages>
 	  <lang>ne</lang>
diff --git a/lohit-master/odia/io.pagure.lohit.odia.font.metainfo.xml b/lohit-master/odia/io.pagure.lohit.odia.font.metainfo.xml
index cb15d2b..3dcb7b2 100644
--- a/lohit-master/odia/io.pagure.lohit.odia.font.metainfo.xml
+++ b/lohit-master/odia/io.pagure.lohit.odia.font.metainfo.xml
@@ -17,9 +17,9 @@
   </description>
   <url type="homepage">https://pagure.io/lohit</url>
   <url type="bugtracker">https://github.com/pravins/lohit/issues</url>
-  <updatecontact>psatpute_at_redhat_dot_com</updatecontact>
+  <update_contact>psatpute_at_redhat_dot_com</update_contact>
   <provides>
-	  <font>Lohit Odia Regular</font>
+	  <font>Lohit Odia</font>
   </provides>
   <languages>
 	  <lang>or</lang>
diff --git a/lohit-master/tamil-classical/io.pagure.lohit.tamil.classical.font.metainfo.xml b/lohit-master/tamil-classical/io.pagure.lohit.tamil.classical.font.metainfo.xml
index 60f23ce..1e77373 100644
--- a/lohit-master/tamil-classical/io.pagure.lohit.tamil.classical.font.metainfo.xml
+++ b/lohit-master/tamil-classical/io.pagure.lohit.tamil.classical.font.metainfo.xml
@@ -17,9 +17,9 @@
   </description>
   <url type="homepage">https://pagure.io/lohit</url>
   <url type="bugtracker">https://github.com/pravins/lohit/issues</url>
-  <updatecontact>psatpute_at_redhat_dot_com</updatecontact>
+  <update_contact>psatpute_at_redhat_dot_com</update_contact>
   <provides>
-	  <font>Lohit Tamil Classical Regular</font>
+	  <font>Lohit Tamil Classical</font>
   </provides>
   <languages>
 	  <lang>ta</lang>
diff --git a/lohit-master/tamil/io.pagure.lohit.tamil.font.metainfo.xml b/lohit-master/tamil/io.pagure.lohit.tamil.font.metainfo.xml
index 22f9054..dcb1663 100644
--- a/lohit-master/tamil/io.pagure.lohit.tamil.font.metainfo.xml
+++ b/lohit-master/tamil/io.pagure.lohit.tamil.font.metainfo.xml
@@ -17,9 +17,9 @@
   </description>
   <url type="homepage">https://pagure.io/lohit</url>
   <url type="bugtracker">https://github.com/pravins/lohit/issues</url>
-  <updatecontact>psatpute_at_redhat_dot_com</updatecontact>
+  <update_contact>psatpute_at_redhat_dot_com</update_contact>
   <provides>
-	  <font>Lohit Tamil Regular</font>
+	  <font>Lohit Tamil</font>
   </provides>
   <languages>
 	  <lang>ta</lang>
diff --git a/lohit-master/telugu/io.pagure.lohit.telugu.font.metainfo.xml b/lohit-master/telugu/io.pagure.lohit.telugu.font.metainfo.xml
index c63a8f9..bfbd609 100644
--- a/lohit-master/telugu/io.pagure.lohit.telugu.font.metainfo.xml
+++ b/lohit-master/telugu/io.pagure.lohit.telugu.font.metainfo.xml
@@ -17,9 +17,9 @@
   </description>
   <url type="homepage">https://pagure.io/lohit</url>
   <url type="bugtracker">https://github.com/pravins/lohit/issues</url>
-  <updatecontact>psatpute_at_redhat_dot_com</updatecontact>
+  <update_contact>psatpute_at_redhat_dot_com</update_contact>
   <provides>
-	  <font>Lohit Telugu Regular</font>
+	  <font>Lohit Telugu</font>
   </provides>
   <languages>
 	  <lang>te</lang>